pkg/kubelet/kubelet_pods.go

```diff
@@ -138,6 +139,11 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 			return nil, err
 		}
 		if mount.SubPath != "" {
```

Comment:
@jhorwit2 did you ever track down whether we need to be concerned with mount.MountPath?